tcf_website/views/calendar.py

```
@@ -0,0 +1,146 @@
+        },
+    )
+
+
+def event_detail(request, event_uri):
+    """Display detailed information for a specific event"""
+    if not getattr(settings, "ENABLE_CLUB_CALENDAR", True):
+        return render(request, "calendar/disabled.html")
+
+    try:
+        event_data = presence.get_event_details(event_uri)
+    except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)